test/battle/hold_effect/choice.c

```diff
@@ -0,0 +1,181 @@
+#include "global.h"
```

Comment:
This folder is usually for battle tests, I would put it with other ai battle tests for now, and then we could make a seperate PR to create a new AI folder for ai-related tests.

Reply:
Super on board for that idea! :D